UPGRADE.md

```diff
@@ -679,3 +679,14 @@ If you use complex where condition directives, such as `@whereConditions`,
 upgrade `mll-lab/graphql-php-scalars` to v4:
 
     composer require mll-lab/graphql-php-scalars:^4
+
+### Schema caching v1 removal
+
+**Likelihood Of Impact: Low**
+
```

Comment:
![we-dont-do-that-here-black-panther](https://user-images.githubusercontent.com/12158000/221150820-8d174d1e-6c64-4254-9c9d-647db4d67d6b.gif)

... or add it everywhere 😉